docs/comparisons.rst

```diff
@@ -0,0 +1,132 @@
+.. highlight:: python
+
+Comparisons
+===========
+
+How does marbles compare to other Python testing tools?
+
+For now, we focus on `pytest`_, which has the most in common with
+marbles.
+
+.. toctree::
+   :maxdepth: 2
+
+
+pytest
+------
+
+Marbles provides some functionality similar to `pytest`_:
+
+* Full statement source code presentation
+* Presentation of variables present in assertion statements
+
+Marbles goes beyond pytest with:
+
+* Note annotations
+* Custom assertion mixins
```

Comment:
We also show more local variables than just those that are used in the actual assertion, allow the author to control which locals are shown, and logging